weblate/trans/tests/test_commands.py

```diff
@@ -345,7 +347,10 @@ def test_import_mercurial_mixed(self):
 
 
 class BasicCommandTest(SimpleTestCase):
-    allow_database_queries = True
+    databases = '__all__'
+
+    if not re.match(r"3.[0-9].[0-9]", django.get_version()):
+        allow_database_queries = True
```

Comment:
Does this have to be behind if? The additional attribute should not matter and this just brings complex statement into several places...

Reply:
Agree with you regarding the complexity but without the "if" we cannot get rid of the Warnings.